flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/nodes/physical/stream/StreamPhysicalWindowAggregate.scala

```diff
@@ -104,14 +105,50 @@ class StreamPhysicalWindowAggregate(
 
   override def translateToExecNode(): ExecNode[_] = {
     checkEmitConfiguration(FlinkRelOptUtil.getTableConfigFromContext(this))
-    new StreamExecWindowAggregate(
-      grouping,
-      aggCalls.toArray,
-      windowing,
-      namedWindowProperties.toArray,
-      InputProperty.DEFAULT,
-      FlinkTypeFactory.toLogicalRowType(getRowType),
-      getRelDetailedDescription
-    )
+    windowing.getWindow match {
+      case windowSpec: SessionWindowSpec =>
+        windowing match {
+          case timeWindowStrategy: TimeAttributeWindowingStrategy =>
+            val (timeAttributeFieldName, timeAttributeIdx) = if (timeWindowStrategy.isRowtime) {
```

Comment:
branch here is not necessary, proctime case will be processed properly inside window op